src/mito2/src/compaction.rs

```diff
@@ -176,8 +175,7 @@ impl CompactionScheduler {
     /// If the region has nothing to compact, it removes the region from the status map.
     fn schedule_compaction_request(&mut self, request: CompactionRequest) -> Result<()> {
         // TODO(hl): build picker according to region options.
```

Comment:
Good job, why not fix `TODO` too?

Reply:
Already fixed. I forgot to remove the todo.